doc/nrf/ug_matter_overview_commissioning.rst

```diff
@@ -0,0 +1,157 @@
+.. _ug_matter_overview_commissioning:
+.. _ug_matter_network_topologies_commissioning:
+
+Matter network commissioning
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The commissioning in Matter is a process of joining a device to a fabric (operational network).
+
+Commissioning device roles
+**************************
+
+The process takes place between the following devices:
+
+* Commissioner device, which is the Matter controller that is carrying out the commissioning.
+* Commissionee device, which is the Matter accessory device that is being added to the fabric.
+
+Commissioning prerequisites
+***************************
+
+To start the commissioning procedure, the controller must get the onboarding information from the Matter accessory device.
+The data payload includes the following information, among other things:
+
+* 16-bit Vendor ID and 16-bit Product ID
+* 12-bit device discriminator
+* 27-bit setup passcode
+* 8-bit Discovery Capabilities Bitmask
+
+The data can be encoded in one of the following ways, both included on-device or in the packaging for the end products:
+
+* Within Manual Pairing Code.
+* Within QR Code.
+  For testing purposes, this code can be printed to the UART console or be shared using an NFC tag.
+
+Manual Pairing Code must always be supported, while the QR Code is recommended.
+Different ecosystems can also define their own methods of providing the discriminator and setup passcode.
+
+Commissioning stages
+********************
+
+The commissioning procedure consists of the stages presented on the following figure.
+Each is discussed in the following sections.
+
+.. figure:: images/matter_network_commissioning.svg
+   :alt: Matter commissioning overview
+
+   Matter commissioning overview
+
+Device discovery
+================
+
+The commissioner discovers devices that can be commissioned onto the network.
+
+The devices that are to be commissioned need to let the commissioner know about their existence, that is *advertise* their presence.
+This can happen using the following advertising methods:
+
+* Bluetooth® LE - This method is used especially if the node is being added to its first Matter fabric.
+* Wi-Fi Access Point - This method uses the Wi-Fi Soft-AP functionality and is used for devices that are not yet members of any network.
```

Comment:
This is out of scope for Matter 1.0

Reply:
Should I leave this with a note "planned" or remove this?